boilerplate/app/models/AuthenticationStore.ts

```diff
@@ -38,6 +38,9 @@ export const AuthenticationStoreModel = types
     setAuthPassword(value: string) {
       store.authPassword = value.replace(/ /g, "")
     },
+    logout() {
+      store.authToken = undefined
+    },
```

Comment:
[q] do we need to also clear `authEmail` and `authPassword`?

Reply:
Yeah we could do that - we didn't before but the `useEffect` in `LoginScreen` to prepopulate should run again since that screen would get mounted again with the navigator being conditional on `isAuthenticated` (which boils down to !!authToken)

Reply:
Updated!